webextensions/api/menus.json

```diff
@@ -292,7 +292,7 @@
             "support": {
               "chrome": {
                 "alternative_name": "contextMenus.ItemType",
-                "version_added": "≤58"
+                "version_added": "44"
```

Comment:
✅ Plausible: https://github.com/chromium/chromium/commit/00f1fc22bfcc653b47d62778170af3eee9855720?diff=split#diff-958be29b1ad371ae427c25879f646d2a170ee1e1305ca77eeabc7098a3fafc67R53

This commit is in [44.0.2359.0](https://github.com/chromium/chromium/releases/tag/44.0.2359.0), and Chrome 44 is based on 44.0.2403 (_after_ that).